src/main/kotlin/ro/hibyte/betting/mapper/BetMapper.kt

```diff
@@ -0,0 +1,33 @@
+package ro.hibyte.betting.mapper
+
+import org.springframework.stereotype.Component
+import ro.hibyte.betting.dto.BetDTO
+import ro.hibyte.betting.entity.Bet
+import ro.hibyte.betting.entity.UserProfile
+import ro.hibyte.betting.repository.UserProfileRepository
+import java.lang.RuntimeException
+import java.util.*
+
+@Component
```

Comment:
Please change this to `@Service`
```suggestion
@Service
```